src/deploy-cromwell-on-azure/Deployer.cs

```diff
@@ -2860,7 +2865,7 @@ void ValidateHelmInstall(string helmPath, string featureName)
             ThrowIfProvidedForUpdate(configuration.BatchAccountName, nameof(configuration.BatchAccountName));
             ThrowIfProvidedForUpdate(configuration.CosmosDbAccountName, nameof(configuration.CosmosDbAccountName));
             ThrowIfProvidedForUpdate(configuration.CrossSubscriptionAKSDeployment, nameof(configuration.CrossSubscriptionAKSDeployment));
-            ThrowIfProvidedForUpdate(configuration.ProvisionPostgreSqlOnAzure, nameof(configuration.ProvisionPostgreSqlOnAzure));
+            //ThrowIfProvidedForUpdate(configuration.ProvisionPostgreSqlOnAzure, nameof(configuration.ProvisionPostgreSqlOnAzure));
```

Comment:
Updating on the AKS path was broken because of this setting and ValidateInitialCommandLineArgs being called twice. If AKS is true, ProvisionPostgreSqlOnAzure is set to true which failed the validation second time this was called. I just commented out for now, since we should be removing this flag and all the AKS flags soon.

Reply:
Merge current `develop` branch into this PR. `UseAks` has already been removed from `Configuration`